src/content/blog/2024/office_hours.mdx

```diff
@@ -0,0 +1,58 @@
+headerImageAlt: 'Person using macbook pro and holding cappuccino'
+announcement:
+    text: 'Introducing nf-core office hours'
+    start: 2024-01-12T16:14:00+01:00
+    end: 2024-01-13T16:14:00+01:00
+pubDate: 2024-01-12T16:14:00+01:00
+authors:
+    - 'christopher-hakkaart'
+label:
+    - 'community'
+---
+
+# nf-core office hours
+
+Slack is great, but sometimes you can't beat sitting down and chatting to someone about your code over a cup of coffee.
+We are thrilled to introduce _nf-core office hours_, weekly drop-in sessions designed for nf-core developers seeking an opportunity to connect and chat.
+
+# What you can expect
+
+nf-core office hours will be held online at regular times, open to anyone to drop in and out as they'd like.
+It can be nice to just sit with some other people working on similar stuff (even virtually).
+And of course it's a great opportunity to chat with others about Nextflow and nf-core development.
+
+We expect that nf-core office hours will be most useful for “advanced beginners” and up - people who have grasped the fundamentals of writing nextflow pipelines, and are now trying to make the leap to designing, coding, and contributing nf-core components and pipelines.
+Please note that it's not designed to be training or mentorships (we have programs for both of those) and there is no guarantee of who will be there and how much help they'll be able to give you.
+However, regardless of what stage you are at, if you think that spending some time talking about developing with nf-core sparks your interest, then the office hours might be just what you're looking for!
+
+In these sessions, some of the nf-core maintainer team and other seasoned community members will be available to engage in discussions about your code.
+They will hang out in Gather during designated hours every week, ready to provide guidance and insights into your nf-core code development.
+[Gather](https://gather.town/) is an online collaborate platform, kind of like Pokemon-meets-Zoom.
+You can get an invite to the nf-core Gather space by going to the `#gather-town` channel on the nf-core Slack, writing a message and dropping the Gather emoji - you'll get an automated message with details.
+For more information, see the [Gather Town bytesize talk](/events/2022/bytesize-37-gathertown).
+
+Office hour sessions won't be structured.
+Some example of things that might be discussed include:
+
+-   A specific concept doesn't make sense and you'd like to talk it over
+-   You're working on a component or pipeline, you're stuck, and you'd like to get unstuck
+-   You'd like to swap code reviews with someone to get your PR merged
+-   You're excited about code and your colleages are tired of hearing about it
+
+# Schedule
+
+We are starting a 10 week trial of _nf-core office hours_ while we evaluate the program.
+We will focus on the European and American time zones initially.
+
+-   TBD
+-   TBD
+-   TBD
```

Comment:
Poll opened, I will give until middle/end of the week before deciding on a day/time.